shared/AppInsightsCore/src/JavaScriptSDK.Interfaces/IExceptionConfig.ts

```diff
@@ -20,6 +43,14 @@ export interface IExceptionConfig{
      *
      * @property {number} maxLogs - Specifies the maximum number of logs that can be generated. If not explicitly set, it defaults to 50.
      */
-    expLog: () => { logs: string[] },
-    maxLogs: number
+    expLog?: () => { logs: string[] },
+
+    /**
+     * The maximum number of logs to include in the telemetry data.
+     * If not explicitly set, it defaults to 50.
+     * This is used in conjunction with the `expLog` callback.
+     * @type {number}
```

Comment:
We don't need either of these 2, typedoc takes care of identifying this as a member of this interface and it's type